Makefile

```diff
@@ -1,3 +1,13 @@
-.PHONY: test
-test:
-	./tests.sh
+KONG_BUILD_TOOLS?=be2c7b3
```

Comment:
This gets removed once Kong/kong-build-tools#21 is merged, correct?

Reply:
yup I should have explicitly called that out thanks 👍

Reply:
done in be3918f